### ISSUE TITLE
string template false positive

### ISSUE THREAD

**Issue by g-back** [abaplint/abaplint#783]
```abap
gp_amount_sp = |{ amount_sp DECIMALS = 2 SIGN = LEFT }|.
```
